rllib/core/rl_trainer/tests/test_trainer_runner.py

```diff
@@ -197,6 +197,44 @@ def test_add_remove_module(self):
             runner.shutdown()
             del runner
 
+    def test_async_update(self):
+        """Test that async style updates converge to the same result as sync."""
+        fws = ["tf", "torch"]
+        # block=True only needs to be tested for the most complex case.
+        # so we'll only test it for multi-gpu-ddp.
+        scaling_modes = ["multi-gpu-ddp"]
+        test_iterator = itertools.product(fws, scaling_modes)
+
+        for fw, scaling_mode in test_iterator:
+            print(f"Testing framework: {fw}, scaling mode: {scaling_mode}.")
+            env = gym.make("CartPole-v1")
+            scaling_config = REMOTE_SCALING_CONFIGS[scaling_mode]
+            runner = get_trainer_runner(fw, env, scaling_config)
+            reader = get_cartpole_dataset_reader(batch_size=500)
```

Comment:
nit: always use batch_size of power of 2. a little background story: https://twitter.com/karpathy/status/1621578354024677377 :)

Reply:
LMAO

Reply:
done